addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -544,7 +544,10 @@ export class HtmlField extends Component {
         }]));
     }
     _onWysiwygBlur() {
-        this.commitChanges();
+        // avoid save on blur if the html field is in inline mode
+        if (!this.props.isInlineStyle) {
+            this.commitChanges();
```

Comment:
I'll explain more in the commit message. but basicaly "breaking the edition" in this context mean : We loose the current cursor position if we save and inline during blur which is a problem if the blur occurred from a edition process ( like dynamic placeholder or media modal) because those process will need the correct range position to insert their result into the document.

Reply:
> can only happen in mass_mailing

Funnily enough it doesn't happens in mass mailing because massmailing inlining result do not override the raw content and thus the editor is still aware of the current cursor position when we refocus in the editor in Javascript.

The problem occured in email template.

Reply:
I said mass_mailing but I meant "context which calls toInline", you're right.